CONTRIBUTING.md

```diff
@@ -37,6 +37,10 @@ To propose a new topic page, please **open an pull request** with your proposed
 
 **Please fill out the pull request template completely.** If you do not fill out the template, your pull request will be closed.
 
+## Guidelines
+
+* Avoid conflicts of interest. If a topic is popular enough to warrant inclusion, someone else will add or improve it.
```

Comment:
Should "conflicts of interest" be elaborated, maybe with an example?

Reply:
Could link out to https://en.wikipedia.org/wiki/Conflict_of_interest, was trying to keep it brief. Open to suggestions or direct edits.